idea-plugin/src/test/kotlin/arrow/meta/ide/plugins/proofs/repo/GradleTypeproofsTest.kt

```diff
@@ -17,8 +17,8 @@ class GradleTypeproofsTest : GradleTestSetUp() {
       val arrowProjectDir = myProjectRoot.findChild("arrow-typeproofs")
       myProjectRoot = arrowProjectDir
 
-      // val checkoutResult = gitCheckout(myProject, "<other-branch>", myProjectRoot)
-      // assertTrue(checkoutResult.success())
+       val checkoutResult = gitCheckout(myProject, "is-fix-ide-errors", myProjectRoot)
```

Comment:
This branch should exist in `arrow-typeproofs` repository.

The test download that repository in the previous statements.